package.json

```diff
@@ -77,7 +77,8 @@
     "test:no-deprecated": "grunt test:prepare && grunt custom:-deprecated && grunt karma:main",
     "test:slim": "grunt test:prepare && grunt custom:slim && grunt karma:main",
     "test": "npm run test:slim && npm run test:no-deprecated && grunt && grunt test:slow && grunt karma:main && grunt karma:esmodules && grunt karma:amd",
-    "jenkins": "npm run test:browserless"
+    "jenkins": "npm run test:browserless",
+    "commitmsg": "commitplease .git/COMMIT_EDITMSG"
```

Comment:
I think I'd prefer not going through npm scripts for some things and the `husky.hooks` field below for others. Moving the command should not be necessary, just fixing the command, I think.

Whatever you try, please verify locally that the check properly fires.

Reply:
Sorry, I've tested and in fact we just need change the hook command, there is no why change to npm script. Thank you!